api/MediaRecorderErrorEvent.json

```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "MediaRecorderErrorEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaRecorderErrorEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "57.0"
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "ie_mobile": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "error": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaRecorderErrorEvent/error",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "57.0"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": "57.0"
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            }
+          }
```

Comment:
Need the "status" block (attribute is on the standard track)

Reply:
Again, there was no status block on the stuff I copied from, so I assumed it was only needed if something was not standard.

Reply:
According to the schema, no status block is allowed here. At least, the schema validation built into VS Code is rejecting it with "Property 'status' is not allowed here."